src/cbc_sdk/connection.py

```diff
@@ -436,25 +436,6 @@ def __init__(self, *args, **kwargs):
                                   pool_maxsize=pool_maxsize,
                                   pool_block=pool_block)
 
-    def raise_unless_json(self, ret, expected):
```

Comment:
This is "dead code" that was removed, so I didn't have to deal with its somewhat squinky use of `ServerError`. It wasn't being used by anything except its test function, which was also removed.

Reply:
Makes sense to me